modules/macstadium_infrastructure/wjb.tf

```diff
@@ -44,5 +44,32 @@ resource "vsphere_virtual_machine" "wjb" {
   }
 }
 
+resource "null_resource" "worker" {
+  triggers {
+    host_id = "${vsphere_virtual_machine.wjb.uuid}"
+    ssh_key_file_signature = "${sha256(file(var.vm_ssh_key_path))}"
+  }
+
+  connection {
+    host = "${vsphere_virtual_machine.wjb.network_interface.0.ipv4_address}"
+    user = "${var.ssh_user}"
+    agent = true
+  }
+
+  provisioner "file" {
+    source      = "${var.vm_ssh_key_path}"
+    destination = "/tmp/travis-vm-ssh-key"
+  }
+
+  provisioner "remote-exec" {
+    inline = [
+      "if ! getent passwd travis-worker >/dev/null; then sudo useradd -r -s /usr/bin/nologin travis-worker; fi"
```

Comment:
This has a race condition since other provisioners might create the user between `getent` and `useradd`, but the same code is used elsewhere so I'm not too worried. We could always add some `|| getent passd travis-worker >/dev/null` again after the `useradd` if it becomes a problem.